src/components/AddEvent.tsx

```diff
@@ -181,29 +193,65 @@ const AddEvent = () => {
         <Form.Label>Date:</Form.Label>
         <DateInput />
       </Form.Group>
-      <Form.Group>
-        <Form.Check
-          name="group"
-          type="radio"
-          id="30"
-          onChange={checkChangeHandler}
-          label="Repeat for 30 days"
-        />
-        <Form.Check
-          name="group"
-          type="radio"
-          id="60"
-          onChange={checkChangeHandler}
-          label="Repeat for 60 days"
-        />
-        <Form.Check
-          name="group"
-          type="radio"
-          id="90"
-          onChange={checkChangeHandler}
-          label="Repeat for 90 days"
-        />
-      </Form.Group>
+      <div className="recurrence">
```

Comment:
Why this div

Reply:
For styling purposes. I`m giving display: flex to the div with class of "recurrence" so that the two form groups are positioned inline.

- No recurrence selected

<img width="461" alt="No recurrence selected" src="https://user-images.githubusercontent.com/96878439/226592079-78b87f8e-3440-40a1-a24c-f2e916cafbf9.png">

- Recurrence selected

<img width="425" alt="Recurrence selected" src="https://user-images.githubusercontent.com/96878439/226592083-ba162e1d-a4d5-425b-802b-6086dd725ae1.png">

- Recurrence selected without display:flex on recurrence class

<img width="473" alt="Recurrence selected without display - flex" src="https://user-images.githubusercontent.com/96878439/226592084-a64ac094-589b-45c9-8ea1-ede46896e0f6.png">

Reply:
thanks for the extensive comment :)